NuRadioReco/framework/station.py

```diff
@@ -46,6 +56,9 @@ def get_channel_ids(self):
     def add_channel(self, channel):
         self.__channels[channel.get_id()] = channel
 
+    def remove_channel_id(self, channel_id):
```

Comment:
shouldn't this function be called "remove_channel"? 

Also please add docstrings at least to the new functions. Github Copilot can often autogenerate them.

Reply:
I think the reasoning here was that seeing the function accepts a channel ID rather than a Channel object, it was better named `remove_channel_id`. I have changed the function to both work with a Channel ID (ie. an integer) or a Channel object itself, and renamed to function `remove_channel`